proto/def/contract.proto

```diff
@@ -16,9 +16,35 @@ message Filter {
   map<string, string> attributes = 1;
 }
 
+// BackoffPolicyType is the type for backoff policies
+enum BackoffPolicy {
+
+  // Exponential backoff policy
+  Exponential = 0;
+
+  // Linear backoff policy
+  Linear = 1;
+}
+
 message EgressConfig {
   // Dead letter is where the event is sent when something goes wrong
   string deadLetter = 1;
+
+  // retry is the minimum number of retries the sender should attempt when
+  // sending an event before moving it to the dead letter sink.
+  uint32 retry = 2;
```

Comment:
no retries is identified as `retry = 0` right?

Reply:
Done: https://github.com/knative-sandbox/eventing-kafka-broker/pull/258/commits/7bba73d648a70af57f9a88ab259ff472415b9ef0